dotcom-rendering/src/components/ArticleBody.tsx

```diff
@@ -63,7 +68,7 @@ const globalOlStyles = () => css`
 			${body.medium({ lineHeight: 'tight' })};
 			content: counter(li);
 			counter-increment: li;
-			margin-right: ${space[1]}px;
+			margin-right: ${remSpace[1]}px;
```

Comment:
Don't think we need 'px' or 'rem' after [remspace](https://github.com/guardian/source/blob/d84c25d67ed51545994f53986eb4487d35c1aab4/packages/%40guardian/source-foundations/src/space/space.ts#L48)

Reply:
thanks I was just fixing that :)